src/pages/edit_resource/ResourceForm.tsx

```diff
@@ -52,6 +52,15 @@ const StyledButtonWrapper = styled.div`
   padding: 2rem 2rem 1rem;
 `;
 
+const StyledFormStatusWrapper = styled.div`
```

Comment:
Hva synes du om å måtte ha "Styled"-prefix på alle Styled Components ? Jeg Føler det sjelden er nødvendig. Med mindre man skal style spesifikke komponenter som f.ex StyledButton  - kanskje ?

Reply:
Liker Styled prefixet for min del. Indikerer kjapt at det er noe styled-components greier i JSX'en. Men ikke en stor greie hvis du vil ha det bort.